modules/experimental-layers/src/utils/gpu-grid-aggregator.js

```diff
@@ -37,6 +49,70 @@ void main(void) {
 }
 `;
 
+const AGGREGATE_TO_GRID_VS_FP64 = `\
```

Comment:
Nit: large shaders, maybe put in separate files?

Reply:
Sure, I will move them in followup PR.